src/components/HomePage.js

```diff
@@ -0,0 +1,55 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  fetchCountries,
+} from '../redux/countries/countriesSlice';
+import Countries from './Countries';
+import europe from '../images/europe.png';
+
+const Home = () => {
+  const [countryName, searchCountryName] = useState('');
+  const dispatch = useDispatch();
+  const countriesArr = useSelector((state) => state.countries.countriesData);
+  // eslint-disable-next-line max-len
+  const filteredData = countriesArr.filter((country) => country.name.toLowerCase().includes(countryName.toLowerCase()));
```

Comment:
- [ ] Good job removing one of the disabled linter lines @coker2019, but as suggested by the previous reviewer `let's enable all ESLint checks`. This includes this line of `eslint-disable`. Please be reminded linters are here to help us write clean code.

Reply:
This change has not yet been addressed.